src/game/utils/settings.c

```diff
@@ -25,29 +25,29 @@
 static settings _settings;
 static const char *settings_path;
 
-typedef enum field_type_t
+typedef enum
 {
     TYPE_INT,
     TYPE_FLOAT,
     TYPE_BOOL,
     TYPE_STRING
 } field_type;
 
-typedef union field_default_u {
+typedef union {
     char *s;
     int i;
     int b;
     double f;
 } field_default;
 
-typedef struct field_t {
+typedef struct {
```

Comment:
struct variables sorted for reduced padding (and memory use). Not a big thing, but might as well.